install-appliance.md

```diff
@@ -70,15 +95,30 @@ In this example, we use three storage volumes:
 
 If you don't have any specific requirements, we recommend choosing the same configuration.
 
-#### Configure the security group
+Click **Next: Add Tags** and then **Next: Configure Security Group** to continue.
+
+#### 6. Configure the security group
 
 To allow external access, you must open several ports in the security group attached to the AWS instance. The AMI already comes with the required configuration, so you don't need to do any changes. For reference, all required ports are documented [here](https://discourse.ubuntu.com/t/requirements/17734).
 
-![03_appliance_security_types|690x534](upload://9KG97kpHpVdvvMDIp7qdQFnO6zT.png)
+<!-- update image -->
+![Configure the security group](images/install_appliance_security-group.png)
+
+Click **Review and Launch** to continue.
 
-#### Review and launch
+#### 7. Review and launch
 
-You should now review the instance configuration. If everything is correct, launch the instance.
+You should now review the instance configuration. If everything is correct, click **Launch**.
+
+You are prompted to select a key pair. You can choose an existing key pair or create one if you don't have one yet. Make sure to save the private key in a secure location.
+
+<!-- add image -->
+![Confirm to launch instances](images/install_appliance_launch-instances.png)
+
+Click **Launch Instances** to continue. AWS will verify your configuration, subscribe you to the product and launch the instance.
```

Comment:
Hasn't subscribing happened way before this?

Reply:
No. If you start without a subscription, you see the prices when you select the AMI (but you just click "continue") and then the real subscribing happens when you launch the instance.

Reply:
Ok